pkg/reconciler/broker/broker.go

```diff
@@ -415,3 +456,65 @@ func (r *Reconciler) reconcileIngressService(ctx context.Context, b *v1alpha1.Br
 	expected := resources.MakeIngressService(b)
 	return r.reconcileService(ctx, expected)
 }
+
+// reconcileTriggers reconciles the Triggers that are pointed to this broker
+func (r *Reconciler) reconcileTriggers(ctx context.Context, b *v1alpha1.Broker, filterSvc *corev1.Service) error {
+
+	// TODO: Figure out the labels stuff... If webhook does it, we can filter like this...
+	// Find all the Triggers that have been labeled as belonging to me
+	/*
+		triggers, err := r.triggerLister.Triggers(b.Namespace).List(labels.SelectorFromSet(brokerLabels(b.brokerClass)))
```

Comment:
wouldn't that be more specific/correct, than `Everything` ?

Reply:
Absolutely :) I wanted to keep the existing behaviour and then add the bits for multiple brokers. Right now this will not change any behaviour, it will reconcile all the Triggers, just like in the past. Cool?